src/firefly_exchange_client/constants.py

```diff
@@ -32,6 +33,7 @@
     "socketURL": "wss://dapi.api.arbitrum-prod.firefly.exchange",
     "webSocketURL": "wss://notifications.api.arbitrum-prod.firefly.exchange",
     "onboardingUrl": "https://trade-arb.firefly.exchange",
+    "UUID": "fc43ae3d-b88b-4db9-8af3-612b1796343a"
```

Comment:
Removed it from defaulting @MaazUddin11 , made it "uuid-default". And also made the code backwards compatible even if the UUID parameter is missing, it would work.